### PR TITLE
feat: Add math functions and usage examples

### PR DESCRIPTION
**Pull request by agupta006**
## Description
This PR introduces basic math functions and usage examples to the project.

### Changes Made:
- New file app.js added with basic math functions (sum, multiply, subtract, divide)
- Added usage examples for all math functions
- New .env file with commented environment variables
- Updated VS Code settings

### New Features
- Introduces basic mathematical operations module with four core functions:
  - `sum(a, b)`: Addition operation
  - `multiply(a, b)`: Multiplication operation
  - `subtract(a, b)`: Subtraction operation (note: currently misspelled as "substract")
  - `divide(a, b)`: Division operation

### Dependencies
- No new dependencies added
- Pure JavaScript implementation

### Breaking Changes
- None - This is a new module with no existing consumers

### Performance Impact
- Minimal - simple arithmetic functions with O(1) complexity
- No loops or recursive operations

### Security Considerations
- **Critical**: Division by zero not handled in divide function
- Could lead to `Infinity` or `-Infinity` results
- Recommendation: Add input validation

### Testing Requirements
- Unit tests needed for all math functions:
  - Test normal cases
  - Test edge cases (zero, negative numbers)
  - Test division by zero scenario
  - Test with non-numeric inputs

### Additional Notes
- Function naming inconsistency: "substract" should be "subtract"
- Only one calculation result is logged to console
- .env file missing newline at end of file

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+function sum(a, b) {
+  return a + b;
+}
+
+function multiply(a, b) {
+  return a * b;
+}
+
+function substract(a, b) {
```
Comment (agupta006) [inline app.js:9]: Typo: Should be "subtract" not "substract"

File: app.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  return a - b;
+}
+
+function divide(a, b) {
+  return a / b;
```
Comment (agupta006) [inline app.js:14]: Security: Add validation to prevent division by zero

File: app.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+const sumVal = sum(2, 3);
+const mulVal = multiply(4, 5);
+const subVal = substract(10, 4);
+const divVal = divide(20, 5);
+console.log(sumVal);
```
Comment (agupta006) [inline app.js:21]: Suggestion: Consider logging all calculated values, not just sumVal

File: .env [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# SAGE_ENV=dev
+# ALLOW_DEV_MODE=true
```
Comment (agupta006) [inline .env:2]: Note: Remember to add a newline at the end of the file

**Review by agupta006 — COMMENTED** [review]
## Code Review Summary

I've reviewed the changes in this PR and provided line-specific feedback above. Here's a summary of the key findings:

### Issues Found:
1. **Typo** (Line 9): Function name "substract" should be "subtract"
2. **Security** (Line 14): Division by zero is not handled in the divide function
3. **Code Quality** (Line 21): Only one calculation result is logged

### Recommendations:
- Fix the typo in the function name
- Add input validation for the divide function to handle edge cases
- Consider logging all calculation results for better debugging
- Add a newline at the end of the .env file

Overall, the implementation is clean and straightforward, but these improvements would enhance code quality and robustness.
